ui/raidboss/data/05-shb/raid/e8s.js

```diff
@@ -705,294 +705,295 @@
+        'Rush': 'ラッシュ',
+        'Scythe/Axe Kick': 'サイスキック/アクスキック',
+        'Shattered World': 'シャッタード・ワールド',
+        'Shining Armor': 'ブライトアーマー',
+        'Shock Spikes': 'ショックスパイク',
+        'Skyfall': '世界消滅',
+        'Stoneskin': 'ストンスキン',
+        'Spiteful/Embittered Dance': '冷厳の舞踏技 / 峻厳の舞踏技',
         'the Path of Light': '光の波動',
         'the House of Light': '光の津波',
-        'attack': '攻撃',
+        'Twin Silence/Stillness': '静寂の双剣技 / 静寂の双剣技',
+        'Twin Stillness/Silence': '静寂の双剣技 /静寂の双剣技',
         'Wyrm\'s Lament': '聖竜の咆哮',
-        'Twin Stillness': '静寂の双剣技',
-        'Stoneskin': 'ストンスキン',
-        'Spiteful Dance': '冷厳の舞踏技',
-        'Skyfall': '世界消滅',
-        'Shock Spikes': 'ショックスパイク',
-        'Shining Armor': 'ブライトアーマー',
-        'Shattered World': 'シャッタード・ワールド',
-        'Scythe Kick': 'サイスキック',
-        'Rush': 'ラッシュ',
-        'Reflected Shining Armor': 'ミラーリング・ブライトアーマー',
-        'Reflected Scythe Kick': 'ミラーリング・サイスキック',
-        'Reflected Hallowed Wings': 'ミラーリング・ホーリーウィング',
-        'Reflected Frost Armor': 'ミラーリング・フロストアーマー',
-        'Reflected Drachen Armor': 'ミラーリング・ドラゴンアーマー',
-        'Reflected Biting Frost': 'ミラーリング・フロストスラッシュ',
-        'Morn Afah': 'モーン・アファー',
-        'Mirror, Mirror': '鏡の国',
-        'Longing of the Lost': '聖竜気',
-        'Light Rampant': '光の暴走',
-        'Icicle Impact': 'アイシクルインパクト',
-        'Icelit Dragonsong': '氷と光の竜詩',
-        'Holy': 'ホーリー',
-        'Heavenly Strike': 'ヘヴンリーストライク',
-        'Heart Asunder': 'ハートアサンダー',
-        'Hallowed Wings': 'ホーリーウィング',
-        'Frost Armor(?! )': 'フロストアーマー',
-        'Frigid Water': 'アイスフロスト',
-        'Frigid Stone': 'アイスストーン',
-        'Frigid Needle': 'アイスニードル',
-        'Frigid Eruption': 'アイスエラプション',
-        'Driving Frost': 'フロストスラスト',
-        'Draconic Strike': 'ドラコニックストライク',
-        'Drachen Armor': 'ドラゴンアーマー',
-        'Double Slap': 'ダブルスラップ',
-        'Diamond Frost': 'ダイヤモンドダスト',
-        'Bright Pulse': '閃光',
-        'Bright Hunger': '浸食光',
-        'Biting Frost': 'フロストスラッシュ',
-        'Banish III Divided': 'ディバイデッド・バニシュガ',
-        'Banish III': 'バニシュガ',
-        'Banish Divided': 'ディバイデッド・バニシュ',
-        'Banish(?! )': 'バニシュ',
-        'Axe Kick': 'アクスキック',
-        'Akh Rhai': 'アク・ラーイ',
-        'Akh Morn': 'アク・モーン',
-        'Absolute Zero': '絶対零度',
 
         // FIXME
+        'Inescapable Illumination': 'Inescapable Illumination',
+        'Reflected Armor \\(B\\)': 'Reflected Armor (B)',
+        'Reflected Armor \\(G\\)': 'Reflected Armor (G)',
+        'Reflected Armor \\(R\\)': 'Reflected Armor (R)',
+        'Reflected Drachen': 'Reflected Drachen',
         'Reflected Frost \\(G\\)': 'Reflected Frost (G)',
         'Reflected Frost \\(R\\)': 'Reflected Frost (R)',
         'Reflected Kick \\(G\\)': 'Reflected Kick (G)',
         'Reflected Wings \\(B\\)': 'Reflected Wings (B)',
         'Reflected Wings \\(G\\)': 'Reflected Wings (G)',
         'Reflected Wings \\(R\\)': 'Reflected Wings (R)',
```

Comment:
```javascript
'Inescapable Illumination': '曝露光',
'Reflected Armor \\(B\\)': '反射アーマー（青）',
'Reflected Armor \\(G\\)': '反射アーマー（緑）',
'Reflected Armor \\(R\\)': '反射アーマー（赤）',
'Reflected Drachen': '反射ドラゴンアーマー',
'Reflected Frost \\(G\\)': '反射フロスト（緑）',
'Reflected Frost \\(R\\)': '反射フロスト（赤）',
'Reflected Kick \\(G\\)': '反射キック',
'Reflected Wings \\(B\\)': '反射ホーリーウィング（青）',
'Reflected Wings \\(G\\)': '反射ホーリーウィング（緑）',
'Reflected Wings \\(R\\)': '反射ホーリーウィング（赤）',
```

I think it should be translated like this, but since I have not enter this raid exactly, so I don't know if it is correct enough. I just learn from Youtube videos. (*￣3￣)╭

Reply:
Thank you